driver/sql/projection_notification_processor_test.go

```diff
@@ -0,0 +1,111 @@
+package sql
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+type notificationQueueMock struct {
```

Comment:
Question we already use [`mockgen`](https://github.com/hellofresh/goengine/blob/master/mocks/mocks.go#L11) for the [mocks](https://github.com/hellofresh/goengine/tree/master/mocks) why did you not use this?